### ISSUE TITLE
Q: Is there a way to suppress the average section at the end of the report output?

### ISSUE THREAD

**Issue by Pitterling** [sysstat/sysstat#279]
man page / documentation does not contain any information.

**Comment by sysstat** [comment]
No, sar always displays the average values at the end of the report.
Unlike sar, sadf never displays them in its various output formats.

**Comment by Pitterling** [comment]
hmm .. but sadf is only for already collected data. Usually this is collected for long intervals .. like 10min.
In case of a performance analysis I'm using sar with xx-seconds intervals to get detailed statistics lets say for 30min. I would not be interested in the averages .. but many thx anyway

**Comment by sysstat** [comment]
I don't know what is your need exactly, but you can still use sar with xx-seconds intervals and save its output to a file (using something like `sar -o /my/data/file [...]`) then use sadf to read those data back and do performance analysis.
Or if you don't want to use sar interactively, you can also start the data collector (sadc) with an xx-seconds interval from a crontab entry (e.g. start this command every minute, time interval is 10s : `/path/to/sadc /my/data/file 10 6 [...]`).

**Comment by Pitterling** [comment]
sar collection is part of a broader data collection. The analysis is done on a different server. Therefore each data collector is storing directly in individual text file within a data collection working directory.
storing as binary and re-reading with sadf/sadc would complicated things .. so i stick to my implementation. I just wanted to know if i can avoid the average lines .. Thx a lot!
